### PR TITLE
Add section on how to cite

### LINKED ISSUE #155: Add "How to Cite" section to the docs
The section should cover the following information:
- Terms of use: Explain the Terms of Use relating to citing in human-friendly language. 
From section 3 (i) from our Terms of Use
> publish, present or otherwise disclose Data, or results from analysis of Data, obtained through the ScPCADP without properly attributing (i) the Data source using the language provided by the Data contributor of that Data set (as applicable), and (ii) us using the language posted on the ScPCADP or otherwise provided by us;

- Provide a citation for the portal. 
   - Provide APA and MLA format like https://cellxgene.cziscience.com/docs/08__Cite%20cellxgene%20in%20your%20publications
   - Use the organization name ("Alex's Lemonade Stand Foundation Childhood Cancer Data Lab") instead of listing individual contributor names as we do in the refine.bio footer until we preprint/publish.
   - Tell users that they can find the accessed date in the README (to be added) in the download folder. 

- Provide directions to cite individual projects
  - Pointing people to where to find publications and project descriptions for individual projects
  - If no publication is listed, note that the publication information on the Portal might lag and ask people to contact the project contact for more up-to-date information/what they want people to cite.
  - Remind them they can find the link to the project page of the data in the README in the download folder.

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Issue/Purpose

Closes #155 

Here, I'm adding the initial section on how to cite the Portal.

### Questions/comments for reviewers

* Does the position of the citation section in the IA make sense to you?
* Is the initial sentence summarizing the Terms of Use sufficiently human-friendly?

### CODE REVIEW

File: docs/citation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# How to Cite
+
+Per [the ScPCA Portal Terms of Use](https://scpca.alexslemonade.org/terms-of-use), users must attribute the data source using any applicable language supplied by data submitters and provided by the Portal when publishing or presenting data or results.
```
Comment (jashapiro) [inline docs/citation.md:3]: This deviates a bit more from the wording in the terms of use, but I like the verb `credit` over `attribute` in this case?
```suggestion
Per [the ScPCA Portal Terms of Use](https://scpca.alexslemonade.org/terms-of-use), users must credit the data source using any applicable language supplied by data submitters and provided by the Portal when publishing or presenting data or results.
```

File: docs/citation.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Citing a project
+
+Please review the project page for the project you are citing for any citation language included in the project abstract or listed publications.
```
Comment (jashapiro) [inline docs/citation.md:7]: One more adjective here?
```suggestion
Please review the project page for the project you are citing for any specific citation language included in the project abstract or listed publications.
```

File: docs/citation.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Citing the ScPCA Portal
+
+### APA Format
+
+Alex's Lemonade Stand Foundation Childhood Cancer Data Lab. (n.d.). The Single-cell Pediatric Cancer Atlas Portal. Retrieved (insert access date), from https://scpca.alexslemonade.org/
+
+### MLA Format
+
+The Single-Cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed (insert access date).
```
Comment (jashapiro) [inline docs/citation.md:22]: For consistency, do we want to use handlebars here too?

```suggestion
Alex's Lemonade Stand Foundation Childhood Cancer Data Lab. (n.d.). The Single-cell Pediatric Cancer Atlas Portal. Retrieved {insert access date}, from https://scpca.alexslemonade.org/

### MLA Format

The Single-Cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed {insert access date}.
```

And, just for amusement:
```
### ICMJE Format
The Single-Cell Pediatric Cancer Atlas Portal [Internet]. Wynnewood (PA): Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab. 2022 - [cited {insert access date}]. Available from: https://scpca.alexslemonade.org/
```
(It's the `[Internet]` followed by a physical location for me.)

File: docs/index.md [hunk 4]
```diff
@@ -11,4 +11,5 @@ download_files
 sce_file_contents
 getting_started
 faq
+citation
```
Comment (jashapiro) [inline docs/index.md:14]: I think thinks makes sense as a location. I could see an argument for before FAQ, but I do prefer this.

**Review by jashapiro — APPROVED** [review]
LGTM! A couple minor suggestions and one that I can't decide whether I am serious about.

File: docs/citation.md (hunk 1 above)
Comment (jaclyn-taroni) [reply docs/citation.md:3]: I could go either way, so let's see what @dvenprasad thinks!

**Review by dvenprasad — CHANGES_REQUESTED** [review]
The structure looks good, and all of the information needed seems to be there.

I've left a suggestion to be clear and firm in what we expect users to cite and a note to let people know where to find the access date.

File: docs/citation.md (hunk 1 above)
Comment (jaclyn-taroni) [reply docs/citation.md]: I did not use your exact suggestion here. Instead, I noted that you must cite the individual project(s) and the portal while linking to the relevant sections.

**Comment by jaclyn-taroni** [comment]
This is ready for another look, @dvenprasad. You may still disagree with my take on the intro bullet points, but I didn't want to accidentally imply that users don't need to cite listed pubs.

**Review by dvenprasad — APPROVED** [review]
Looking good. I like the way you have listed it too.
🦜 🦜 🦜

**Pull request merged**